package pl.marceligrabowski.githubtask.external.error.exception

class ExternalServiceUnavailableException : RuntimeException("External service is unavailable") {
}